contracts/BIFI/infra/BeefyVaultRegistry.sol

```diff
@@ -0,0 +1,199 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC20/IERC20Upgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/structs/EnumerableSetUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+interface IUniswapV2Pair {

```

Comment:
please use existing interfaces or import uniswap-lib, otherwise it will conflict and scripts with `getContractAt` will fail.
can also name them specific to this file, `IStrategy/IVault` -> `IBeefyVaultRegistryStrategy/IBeefyVaultRegistryVault`

Reply:
Sounds good, I included them because they wouldn't compile with the current pragma settings.